__tests__/parseCliArgs.test.mjs

```diff
@@ -214,6 +214,35 @@ test('It should return default value for fontUnit if passing invalid value (long
   );
 });
 
+/*
+ * Opacities unit
+ */
+test('It should return the unit for opacitiesUnit if passing valid value (long-hand)', () => {
+  const unit = 'percent';
+  expect(parseCliArgs(['--opacitiesUnit', unit])).toEqual(
+    expect.objectContaining({
+      opacitiesUnit: unit
+    })
+  );
+});
+
+test('It should return the unit for opacitiesUnit if passing valid value (short-hand)', () => {
+  const unit = 'float';
```

Comment:
Would be good to have the new CLI args documented in README

Reply:
Done, you can review :)